compiler-plugin/src/main/java/io/ballerina/stdlib/websub/task/ServiceAnalysisTask.java

```diff
@@ -55,6 +57,10 @@ public void perform(SyntaxNodeAnalysisContext context) {
         if (serviceDeclarationOpt.isPresent()) {
             ServiceDeclarationSymbol serviceDeclarationSymbol = (ServiceDeclarationSymbol) serviceDeclarationOpt.get();
             if (isWebSubService(serviceDeclarationSymbol)) {
+                if (serviceNode.members().isEmpty()) {
+                    WebSubDiagnosticCodes codeActionDiagnostic = WebSubDiagnosticCodes.WEBSUB_202;
+                    updateContext(context, codeActionDiagnostic, serviceNode.location());
+                }
```

Comment:
As we are adding the `SubscriberServiceConfig` annotation and the `onEventNotification` function at the same code action, what happens if someone already has the annotation execute the code action? It'll add another `SubscriberServiceConfig` which will result in a compilation error. So shouldn't we check the presence of the annotation as well?

Reply:
**Just as a suggestion.**

Isn't it good to have two separate code actions for generating the `SubscriberServiceConfig` and another for generating the mandatory `onEventNotification` function?

Reply:
Fixed in [f901fe2dd5f880fc4a6b5e3e544d64e809cb84cd](https://github.com/ballerina-platform/module-ballerina-websub/pull/519/commits/f901fe2dd5f880fc4a6b5e3e544d64e809cb84cd)